src/main/java/com/yugabyte/sample/apps/AppBase.java

```diff
@@ -172,7 +172,6 @@ protected Connection getPostgresConnection(String database, String username, Str
         if (password != null) {
           props.setProperty("password", password);
         }
-        props.setProperty("sslmode", "disable");
```

Comment:
Maybe comment this out and add explanation for why.

Reply:
would prefer to just remove it, given we are overriding a sane default for no apparent reason.